### PR TITLE
Update nidcpower, niswitch from latest exports

### PR DESCRIPTION
**Pull request by dmondrik**
### What does this Pull Request accomplish?

Updates metadata using actual hapigen exports:

`nidcpower_grpc_device 23.0.0d200` (which uses the d196 export of underlying metadata)
`niswitch_grpc_device 23.0.0d67` (which uses the d58 export of underlying metadata)

### Why should this Pull Request be merged?

[Task 2186877](https://dev.azure.com/ni/DevCentral/_workitems/edit/2186877): [grpc-device] Uptake new NI-DCPower Export
[Task 2186882](https://dev.azure.com/ni/DevCentral/_workitems/edit/2186882): [grpc-device] Uptake new NI-SWITCH Export

### What testing has been done?

Built locally.

### CODE REVIEW

File: generated/nidcpower/nidcpower_library.h [hunk 1]
```diff
@@ -118,14 +118,14 @@ class NiDCPowerLibrary : public nidcpower_grpc::NiDCPowerLibraryInterface {
   ViStatus GetExtCalLastDateAndTime(ViSession vi, ViInt32* year, ViInt32* month, ViInt32* day, ViInt32* hour, ViInt32* minute);
   ViStatus GetExtCalLastTemp(ViSession vi, ViReal64* temperature);
   ViStatus GetExtCalRecommendedInterval(ViSession vi, ViInt32* months);
-  ViStatus GetLCRCompensationData(ViSession vi, ViConstString channelName, ViInt32 compensationDataSize, ViInt8 compensationData[]);
+  ViStatus GetLCRCompensationData(ViSession vi, ViConstString channelName, ViInt32 compensationDataSize, ViAddr compensationData[]);
```
Comment (dmondrik) [inline generated/nidcpower/nidcpower_library.h:121]: The proto file is right to say the parameter is `bytes` but this is definitely wrong. It's either a `ViInt8[]` or just a simple scalar `ViAddr`. What do we do to fix this? @DavidCurtiss @phindman @reckenro

Comment (phindman) [reply generated/nidcpower/nidcpower_library.h:121]: Daniel Jaramillo, @alexdubois-ni, and @marcoskirsch discussed how to address the ViInt8[] vs ViAddr inconsistency, and the hapigen plugins were modified in [AzDO PR 377136](https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/377136).

It was noted that some plugin-specific metadata changes might be necessary.  The type of metadata change suggested is to turn the `type` back to `i8` (or the equivalent for that plugin).  This sort of change was made to the CXP-specific metadata [here](https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/379352?_a=files&path=/src/nidcpower/nidcpowerBaseDistribution/nidcpowerAPIMetadata/source/nidcpowerAPIMetadata/cxp/functions/functions.hapigen).

Comment (dmondrik) [reply generated/nidcpower/nidcpower_library.h:121]: Posted a [fix for the gRPC expander](https://ni.visualstudio.com/DevCentral/_git/ni-central/pullrequest/381187).

**Review by astarche — APPROVED** [review]

File: generated/nidcpower/nidcpower_library.h (hunk 1 above)
Comment (dmondrik) [reply generated/nidcpower/nidcpower_library.h]: Also, Ryan's fix yesterday to the expander went in, and this uses it now. The signatures seem correct.

**Pull request merged**